### PR TITLE
Map fields for image rights

### PR DESCRIPTION
**Pull request by ioanaciont**

### CODE REVIEW

File: src/test/java/com/ft/methodeimagemodelmapper/resources/MethodeImageModelResourceComponentTest.java [hunk 1]
```diff
@@ -72,7 +71,9 @@ public void testImageModelComesThrough() throws Exception {
         assertThat(actualContent.getExternalBinaryUrl(), equalTo(expectedContent.getExternalBinaryUrl()));
         assertThat(actualContent.getPublishReference(), equalTo(expectedContent.getPublishReference()));
         assertThat(actualContent.getFirstPublishedDate(), equalTo(expectedContent.getFirstPublishedDate()));
-        assertThat(actualContent.getCanBeDistributed(), equalTo(Distribution.VERIFY));
+        assertThat(actualContent.getCanBeDistributed(), equalTo(expectedContent.getCanBeDistributed()));
+        assertThat(actualContent.getCanBeSyndicated(), equalTo(expectedContent.getCanBeSyndicated()));
+        assertThat(actualContent.getRightsGroup(), equalTo(expectedContent.getRightsGroup()));
```
Comment (iulianp26) [inline src/test/java/com/ft/methodeimagemodelmapper/resources/MethodeImageModelResourceComponentTest.java]: Is there a reason why there is no check for masterSource field?

File: src/test/java/com/ft/methodeimagemodelmapper/resources/MethodeImageModelResourceComponentTest.java [hunk 2]
```diff
@@ -103,7 +104,9 @@ public void testImageModelComesThroughIfJsonNotComplete() throws Exception {
...
```
Comment (iulianp26) [inline src/test/java/com/ft/methodeimagemodelmapper/resources/MethodeImageModelResourceComponentTest.java]: Is there a reason why there is no check for masterSource field?

Comment (ioanaciont) [inline src/test/java/com/ft/methodeimagemodelmapper/resources/MethodeImageModelResourceComponentTest.java]: The only reason is that I forgot about it :)

File: src/test/java/com/ft/methodeimagemodelmapper/resources/MethodeImageModelResourceComponentTest.java (hunk 1 above)
Comment (ioanaciont) [inline src/test/java/com/ft/methodeimagemodelmapper/resources/MethodeImageModelResourceComponentTest.java]: Fixed 👍

File: src/main/java/com/ft/methodeimagemodelmapper/service/MethodeImageModelMapper.java [hunk 3]
```diff
@@ -94,6 +101,26 @@ private boolean isEomTypeSupported(final EomFile eomFile) {
                 copyrightNotice = "© " + copyrightNotice;
             }
 
+            String distributionValue = xpath.evaluate("/meta/picture/FTAggregation", attributesDocument);
```
Comment (tosan88) [inline src/main/java/com/ft/methodeimagemodelmapper/service/MethodeImageModelMapper.java]: Should be `/meta/picture/FTRights/FTAggregation`

**Pull request merged**